.devcontainer.json

```diff
@@ -0,0 +1,25 @@
+{
+  "name": "Node.js Core Developer Environment",
+  "runArgs": [
+    "--platform=linux/amd64"
+  ],
+  "customizations": {
+    "vscode": {
+      "extensions": [
+        "github.vscode-pull-request-github",
+        "ms-vsliveshare.vsliveshare",
+        "vscode-icons-team.vscode-icons",
+        "visualstudioexptteam.vscodeintellicode"
+      ],
+      "settings": {
+        "terminal.integrated.profiles.linux": {
+          "zsh (login)": {
+            "path": "zsh",
+            "args": ["-l"]
+          }
+        }
+      }
+    }
+  },
+  "image": "nodejs/devcontainer:nightly"
```

Comment:
Are people using this? The image hasn't been updated in two years: https://hub.docker.com/r/nodejs/devcontainer

Reply:
I'm not sure, but it's what we've been recommending.